fstest/testserver/init.d/TestSMBKerberos

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE=rclone/test-smb-kerberos
+NAME=smb-kerberos
+USER=rclone
+DOMAIN=RCLONE
+REALM=RCLONE.LOCAL
+SMB_PORT=28633
+KRB5_PORT=28634
+
+. $(dirname "$0")/docker.bash
+
+start() {
+    docker build -t ${IMAGE} - <<EOF
```

Comment:
Neat approach to making a lightly customised docker :-)

Do you think we should be removing the image in stop()? Or is docker clever enough to realise it has the same hash so we won't get one extra image per run?

Reply:
Yes, this will benefit from caching, unless the user either deletes the image, or cleans the build cache.

Reply:
In other words, only the first run on a given docker host will actually need to do anything and the next invocations will be "free". Also we use an alpine image which means what needs to be downloaded and the resulting image are really small.